src/components/AttendantForm.jsx

```diff
@@ -0,0 +1,191 @@
+import React, { useState, useEffect } from "react";
+import { getJobTitles, addAttendant, getAttendants } from "../api";
+import AttendantDisplay from "./AttendantDisplay";
+
+export default function AttendantForm() {
+  const [attendants, setAttendants] = useState([]);
+  const [name, setName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [jobTitle, setJobTitle] = useState("");
+  const [age, setAge] = useState("");
+  const [allJobTitles, setAllJobTitles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  async function submitHandler(e) {
+    e.preventDefault();
+    const submittedData = {
```

Comment:
To be more specific:
```suggestion
    const newAttendant = {
```